README.md

```diff
@@ -0,0 +1,17 @@
+eLife Search
+=============
+
+This project is using elasticsearch to index content of the eLife articles and provide a full-text search for Journal.
+
+Installation
+------------
+
+1. Clone the project `git clone https://github.com/elifesciences/search.git`
+2. Run `docker-compose up --build`
+
+Important : Rename `config.php.dist` on local to `config.php` because config.php is in .gitignore.
```

Comment:
This step should probably go before the docker command.

Reply:
How should we access the search api?

http://localhost:????

Reply:
Added step before docker command.

The search API from what @smoqadam shared with me is not accessible from browser.
We had to remove gearman-ui and opensearch-dashboard due to the fact that if we disable SSL for opensearch, the dashboard also needs to be configured to connect to opensearch without ssl (I don't know if that's possible).